test-data/musicxml-testsuite/76b-TrackBalance.musicxml

```diff
@@ -0,0 +1,518 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same with this file, please move it. 

To avoid too much noise, you could just add 1 file with multiple tracks where each track has multiple volumes and balances. Then use it in both tests.

Reply:
Will do.